docs/DeveloperGuide.md

```diff
@@ -309,28 +309,71 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
-
-**MSS**
-
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
-
-    Use case ends.
-
-**Extensions**
-
-* 2a. The list is empty.
-
-  Use case ends.
-
-* 3a. The given index is invalid.
-
-    * 3a1. AddressBook shows an error message.
-
-      Use case resumes at step 2.
+**System: ArtBuddy (AB)**<br>
+**Use case: Add iteration to a commission**<br>
+**Actor: Artist**<br>
+**Guarantees:**
+* An iteration, with an associated image, will be added to the specified commission only if its creation is successful
+* The new iteration will not be created if the image provided is not valid (eg: it does not exist, or it is not
+  of the correct format)
```

Comment:
Unclear if the format refers to the image format or image file path format.

Reply:
Thanks for pointing this out!